db/migrate/20200115124513_upcase_postcodes_on_application_forms.rb

```diff
@@ -0,0 +1,14 @@
+class UpcasePostcodesOnApplicationForms < ActiveRecord::Migration[6.0]
+  def up
+    execute <<-SQL
+      UPDATE
+        application_forms
+      SET
+        postcode = UPPER(postcode)
```

Comment:
Does this need a `WHERE NOT NULL` or is that fine?

Reply:
Just tested this by doing `ApplicationForm.first.update!(postcode: nil)`, then running the migration. It didn't blow up or complain for me. 

Is this the right way to test?

Reply:
I've tested the SQL on my own machine, where I had 3 records `where postcode is null`. It seems to work!